.travis.yml

```diff
@@ -33,8 +33,8 @@ matrix:
     - name: "x86_64-unknown-linux-gnu (stable)"
       rust: stable
       env: TARGET=x86_64-unknown-linux-gnu
-    - name: "x86_64-unknown-linux-gnu (Rust 1.36.0)"
-      rust: 1.36.0
+    - name: "x86_64-unknown-linux-gnu (Rust 1.38.0)"
+      rust: 1.38.0
```

Comment:
Just bump it straight to the latest stable release (1.49.0). That way we will have more flexibility in the future.

Reply:
Just the CI job or the documented MSRV everywhere?

Reply:
The documented MSRV.